ovos_utils/messagebus.py

```diff
@@ -12,13 +12,18 @@
 from ovos_utils.log import LOG
 from ovos_utils.metrics import Stopwatch
 
+# TODO: Consider moving all of this to `ovos_bus_client`
+
 _DEFAULT_WS_CONFIG = {"host": "0.0.0.0",
                       "port": 8181,
                       "route": "/core",
                       "ssl": False}
 
 
 def dig_for_message():
+    # TODO: Deprecate this wrapped reference in 0.1.0
+    LOG.warning("This reference is deprecated. "
```

Comment:
see comment above, this is used in this package, will produce log spam

Reply:
This is just wrapping an import though, why not just force references to import the actual method and handle a missing dependency?

Reply:
because it should be used if available but otherwise not throw an error and still proceed

ovos-utils can not depend on any other ovos package, it is the base package so it would introduce circular imports, these wrappers are here to avoid the circular import hell (can't be done at top level of module)

Reply:
depending on where this is called an exception may be thrown if its application critical, just worried about log spam if done all the time